packages/base/src/api/client.ts

```diff
@@ -18,24 +18,24 @@ export abstract class BaseApiClient {
     this.apiSecret = params.apiSecret;
   }
 
-  protected async init(): Promise<AxiosInstance> {
```

Comment:
Doing this to avoid having to create an entire new `defender-secrets-client` for one endpoint.